library/src/androidMain/kotlin/com/lightningkite/kiteui/views/RView.android.kt

```diff
@@ -123,10 +125,17 @@ actual abstract class RView(context: RContext) : RViewHelper(context) {
             native.background = value
         }
     protected var backgroundBlock: GradientDrawable? = null
+    private var layoutChangeListenerAdded = false
     protected fun updateCorners() {
+        if (!layoutChangeListenerAdded) {
```

Comment:
Make more efficient

Reply:
Made a change so that the listener will only be added when the corner radius depends on the size of the view.